src/data/resolvers/colony.ts

```diff
@@ -21,10 +29,33 @@ import {
   getColonyAllEvents,
 } from './transactions';
 
+type ReputationOracleAddresses = Address[];
+
+const getColonyMembersWithReputation = async (
+  colonyManager: ColonyManager,
+  colonyAddress: Address,
+  domainId: number,
+): Promise<ReputationOracleAddresses> => {
+  const colonyClient = await colonyManager.getClient(
+    ClientType.ColonyClient,
+    colonyAddress,
+  );
+  const { skillId } = await colonyClient.getDomain(domainId);
+  /*
+   * @TODO This is coming from colonyJS as I forgot to adjust the proper types when
+   * creating this common extension.
+   * I'll put in a PR to fix this at some point
+   */
+  // @ts-ignore
```

Comment:
I've put a PR to fix in this colonyJS: https://github.com/JoinColony/colonyJS/pull/480